bundle/src/main/java/com/adobe/acs/commons/genericlists/impl/GenericListImpl.java

```diff
@@ -39,6 +41,7 @@
 
     public static final class ItemImpl implements Item {
 
+        @SerializedName("text")
```

Comment:
Is this a breaking change? Would it be reasonable to assume someone might be consuming this JSON looking for the title property?
Perhaps we should emit 3 properties in the JSON? `title`, `text`, `value` ... where `title` is around for BC?

Reply:
yea, that is likely safer.  

If you trace the history, the sole purpose of this JSON, as best I can tell, is to support asset metadata schema, which uses text/value. At some point, it got changed to title/value/props, which won't work for asset metadata schema. But if someone is using this json for some other purpose or maybe has customized asset metadata schema to use it as is), then removing the title property would break that.

Anyway, I'll fix it so it's BC, but just giving the context for why I did it the way I did.